core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250425_entity_json_serdes_format.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.31.xsd">
```

Comment:
The triggers change to use `JSON_KEYS(new.attributes, '$.attrs')` instead of `JSON_KEYS(new.attributes)` to reflect the latest serialization format. Note that these triggers are tested in `CompactEntityComponentSpec`.